libbindgen/src/lib.rs

```diff
@@ -168,8 +168,8 @@ impl Builder {
     /// Set the input C/C++ header.
     pub fn header<T: Into<String>>(mut self, header: T) -> Builder {
         let header = header.into();
-        self.options.input_header = Some(header.clone());
-        self.clang_arg(header)
+        self.options.input_header = Some(header);
```

Comment:
Given that the only effect of this function is setting `input_header`, I guess it probably makes sense to assert that `input_header` is `None` before assigning it.